pycbc/events/trigger_fits.py

```diff
@@ -211,3 +212,51 @@ def cdf_fn(x):
         return 1 - cum_fndict[distr](x, alpha, thresh)
     return kstest(vals, cdf_fn)
 
+
+def get_masses(bank, tid):
```

Comment:
I won't hold up merger for this, but this seems an odd function. Do you really save much by avoiding the duplication in this case?